### ISSUE TITLE
List and objects variables references are not resolved correctly in Terraform 0.12

### ISSUE THREAD

**Issue by astorath** [VladRassokhin/intellij-hcl#218]
### Prerequisites
 - [x] Ensure you have latest version of plugin installed
 - [x] Search for possible issue duplicates: #208 

### Installation details
 - [x] IDE version (`Help->About->Copy to Clipboard`)
```
IntelliJ IDEA 2019.1.3 (Community Edition)
Build #IC-191.7479.19, built on May 28, 2019
JRE: 1.8.0_202-release-1483-b58 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.5
```
 - [x] intellij-hcl plugin version (`Settings->Plugins`): `v0.7.2`
 - [x] Terraform version (`terraform -v`):
```
Terraform v0.12.0
+ provider.external v1.1.2
+ provider.google v2.7.0
```

### Terraform Configuration Files
```hcl
variable "test" {
  type = list(object({
    name = string
  }))
  default = []
}
locals {
  s = test[0].name
}
variable "test2" {
  type = object({
    name = string
  })
  default = null
}
locals {
  s2 = test2.name
}
```

### Exception
**Unresolved reference name** error in editor

### Expected Behavior
- proper syntax highlight for complex variables
- should not generate an error
- should display code completion suggestion

### Actual Behavior
![image](https://user-images.githubusercontent.com/6577390/58962778-8c58be80-87b4-11e9-81ad-76ad33762a20.png)

**Comment by VladRassokhin** [comment]
Well, example above is incorrect, references to variables must start with `var.` though in some cases plugin still not resolved them. Fixed in 0.7.3

**Comment by astorath** [comment]
> Well, example above is incorrect, references to variables must start with `var.`.
> Though in some cases plugin still not resolved them. Fixed in 0.7.3

Yeah, sorry, fixed it for the record
